src/gui/layout/qgslayoutitemslistview.cpp

```diff
@@ -115,9 +115,9 @@ void QgsLayoutItemsListView::setCurrentLayout( QgsLayout *layout )
 void QgsLayoutItemsListView::showContextMenu( QPoint point )
 {
   QModelIndex index = indexAt( point );
-  QgsLayoutItem *item = mModel->itemFromIndex( index );
-  if ( !item )
+  if ( !index.isValid() )
     return;
+  QgsLayoutItem *item = mModel->itemFromIndex( index );
```

Comment:
Why don't you leave the original check for safety?
```c++
  if ( !item )
    return;
```

Reply:
If `index.isValid()` then `mModel->itemFromIndex( index )` should return an item. I think it's unreachable code, but I can leave it there, sure!

Reply:
Maybe a single `if ( !mModel ) return;` at the beginning of the method would be better?

Reply:
I agree